Vagrantfile

```diff
@@ -28,11 +28,12 @@ Vagrant.configure("2") do |config|
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine and only allow access
   # via 127.0.0.1 to disable public access
-  # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+  # config.vm.network "forwarded port", guest: 80, host: 8080, id: "nginx"
+  config.vm.network "forwarded_port", guest: 81, host: 8081, id: "website"
```

Comment:
You don't need to add port forwarding because you have assigned the VM an IP address below. You want to keep the IP address and access the site via it's IP address. If a machine has it's own IP then you are not using "localhost" anymore you are directly hitting the IP of the vm. 

You should take this out and put back the private network ip line below.